testing/kyotocabinet/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Andrey L <innerspacepilot@gmx.com>
+# Maintainer:  Andrey L <innerspacepilot@gmx.com>
+pkgname=kyotocabinet
+pkgver=1.2.77
+pkgrel=0
+pkgdesc="Kyoto Cabinet is a library of routines for managing a database."
+url="https://fallabs.com/kyotocabinet/"
+arch="all"
+license="GPL-3.0-or-later"
+makedepends="zlib-dev lzo-dev xz-dev"
+subpackages="$pkgname-dev $pkgname-doc"
+source="https://fallabs.com/kyotocabinet/pkg/kyotocabinet-$pkgver.tar.gz"
+builddir="$srcdir/kyotocabinet-$pkgver"
+
+build() {
+	cd "$builddir"
+	./configure --prefix=/usr --enable-lzo --enable-lzma
```

Comment:
if you don't plan on using the static library add --disable-static here (or delete it later on package() if adding it doesn't work), otherwise create a $pkgname-static subpackage